.storybook/cypress/integration/index.js

```diff
@@ -14,7 +14,7 @@ describe('Storybook Article', () => {
 
   it('each story render panel should not be blank', () => {
     cy.get('ul>li>a').each($a => {
-      cy.wrap($a).click();
+      cy.wrap($a).click({ force: true });
```

Comment:
Context to why this is needed would be good here.

cc @jamesbrumpton Cypress test setup change